js/locales/udata.es.json

```diff
@@ -383,7 +383,7 @@
     "The availability of your distant resources (if any) is crucial for reusers. They trust you for the reliability of these data both in terms of reachability and ease of access.": "The availability of your distant resources (if any) is crucial for reusers. They trust you for the reliability of these data both in terms of reachability and ease of access.",
     "The error identifier is {id}": "El identificador de error es  {id}",
     "The files are being uploaded, if you leave now the upload will be canceled.": "The files are being uploaded, if you leave now the upload will be canceled.",
-    "The identity of this public service public is certified by Etalab": "La identidad de este servicio público está certificada por Etalab",
+    "The identity of this public service is certified by %(certifier)": "La identidad de este servicio público está certificada por %(certifier)",
```

Comment:
Actually, these files will be generated by Transifex so no need to do that manually.